From 0124b1c29b9cfe46d73ae82ce023dd7c5b055744 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Thu, 29 Mar 2018 16:36:12 +0800
Subject: [PATCH 09/10] ghostscript-native:fix disable-system-libtiff

Modify configure to add the check to make sure
ghostscrip could work while system-libtiff is
disabled.

Upstream-Status: Pending

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase to ghostscript 9.25.

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
Signed-off-by: Jagadeesh Krishnanjanappa <jkrishnanjanappa@mvista.com>
---
 configure.ac | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/configure.ac b/configure.ac
index 80a60b1..f3e9efb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1319,6 +1319,7 @@ AC_TRY_COMPILE([], [return 0;],
 CFLAGS=$CGLAGS_STORE
 
 if test x"$SHARE_LIBTIFF" = x"0" ; then
+    if test -e $LIBTIFFDIR/configure; then
       echo "Running libtiff configure script..."
       olddir=`pwd`
       if ! test -d "$LIBTIFFCONFDIR" ; then
@@ -1337,6 +1338,10 @@ if test x"$SHARE_LIBTIFF" = x"0" ; then
 
       echo
       echo "Continuing with Ghostscript configuration..."
+    else
+      AC_MSG_NOTICE([Could not find local copy of libtiff.
+Disabling tiff output devices.])
+    fi
 fi
 
 AC_SUBST(SHARE_LIBTIFF)
-- 
1.8.3.1

